client.go

```diff
@@ -30,15 +26,29 @@ type Config struct {
 	// if it expires.
 	RefreshToken string
 
+	// ApiTLD is the top-level domain (without leading dot) of the data center to use.
+	// See https://www.site24x7.com/help/api/index.html#introduction for options.
+	// Defaults to "com" if not given.
+	ApiTLD string
```

Comment:
Instead of having the ApiTLD in the config I would go for two fields `APIBaseURL` and `TokenURL`. If empty the will get their default value from the `APIBaseURL` and `TokenURL` constants respectively.

IMO this is cleaner and would also allow for better testing in the future as `APIBaseURL` and `TokenURL` could be pointed to the address of a `*httptest.Server`.

Reply:
Changed in https://github.com/Bonial-International-GmbH/site24x7-go/pull/34